### DISCUSSION TITLE
Create a method for converting a base64 image to a file in File helpers

### DISCUSSION THREAD

**Discussion by eloquentcoder** [laravel/framework#49543] [Ideas]
I have an idea where one can create a method where you pass a base64 image file to a method and it converts it to a file in laravel. Also we can also create a utility function where we check if a file is a base 64 file. I can add it to the file facade but I would like to know what the reception to this is before opening a PR.
<img width="589" alt="image" src="https://github.com/laravel/framework/assets/80581084/bca28309-863d-4ac3-a5bb-07a8c046e821">

**Comment by shaedrich** [comment]
I suppose, a `static \Illuminate\Filesystem\Filesystem::fromBase64(): self` would be a nice addition!

**Reply by eloquentcoder** [reply]
Just what i thought! Thanks for the feedback will get on it!

**Reply by shaedrich** [reply]
However, don't get your hopes up too high. PRs tend to get rejected when you don't happen to be a Laravel celebrity.

**Reply by eloquentcoder** [reply]
Oops! Well there's no harm in trying! But i'd try not to get my hopes high!

**Reply by shaedrich** [reply]
Fingers crossed!

**Reply by eloquentcoder** [reply]
Made a pull request already! Fingers crossed

**Reply by eloquentcoder** [reply]
I had to close that pull request because of the numerous issues but I have tested the codebase and I have also removed the data:image as the only data uri segment and made a new pull request. Thanks!

**Reply by shaedrich** [reply]
Any time!
